src/components/utils/registerCustomKeysets.ts

```diff
@@ -0,0 +1,65 @@
+import type {KeysetData} from '@gravity-ui/i18n';
+
+import {i18n} from '../../i18n';
+
+export const registerCustomKeysets = (language: string, data: KeysetData) => {
+    validateCustomKeysets(language, data);
+    i18n.registerKeysets(language, data);
+};
+
+function validateCustomKeysets(language: string, data: KeysetData): void {
+    const trustedData = i18n.data.en;
```

Comment:
We provide complete keysets in English language. So we can compare custom keysets with it.

The code below looks kinda verbose, but IDK how to improve it, and it's very reliable and transparent about what needs to be fixed.

Reply:
Also I think creating your first custom keyset might be daunting, because you'll only get a error one at a time. So maybe in our errors we should also show the templated English keyset. In the end our errors will look like this:
```
Custom keysets 'it' validation error: excess component 'Alert' keys for found ["nonexistent_key","nonexistent_key2"]

English keysets example:
{"Alert":{"label_close":"Close"},"Breadcrumbs":{"label_more":"Show more"},"ClipboardButton":{"startCopy":"Copy","endCopy":"Copied!"},"Dialog":{"close":"Close dialog"},"AvatarStack":{"more":["and {{count}} more","and {{count}} more","and {{count}} more"]},"g-clear-button":{"label_clear-button":"Clear"},"Pagination":{"button_previous":"Previous","button_next":"Next","button_first":"First","label_input-placeholder":"Page #","label_page-of":"of","label_select_size":"Select page size"},"Select":{"label_clear":"Clear","label_show-error-info":"Show popup with error info"},"g-user-label":{"label_remove-button":"Remove"},"PinInput":{"label_one-of":"{{number}} of {{count}}, "},"Table":{"label_empty":"No data","label-actions":"Actions","label-row-select":"Select"},"TableColumnSetupInner":{"button_apply":"Apply","button_reset":"Reset","button_switcher":"Columns"},"withTableSettings":{"label_settings":"Table settings"},"TableColumnSetup":{"button_switcher":"Columns"},"Toaster":{"label_close-button":"Close"}}
```


What do you think?